lib/ReactViews/WelcomeMessage/WelcomeMessage.jsx

```diff
@@ -54,19 +54,12 @@ export const WelcomeMessagePure = ({
   isTopElement,
   viewState
 }) => {
+  const { t } = useTranslation();
   // This is required so we can do nested animations
   const [welcomeVisible, setWelcomeVisible] = useState(showWelcomeMessage);
   const [shouldExploreData, setShouldExploreData] = useState(false);
-  const {
-    WelcomeMessage,
-    WelcomeMessagePrimaryBtn,
-    WelcomeMessageSecondaryBtn,
-    WelcomeMessageDissmissText,
-
-    WelcomeMessagePrimaryBtnClick,
-    WelcomeMessageSecondaryBtnClick
-  } = viewState.terria.language;
-
+  const WelcomeMessagePrimaryBtnClick = undefined;
+  const WelcomeMessageSecondaryBtnClick = undefined;
```

Comment:
Keep these loaded in from overrides, otherwise it will break expected behaviour for maps using this